modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogCommon.ts

```diff
@@ -89,7 +100,7 @@ const renderModalDialog = (spec: DialogSpec, initialData, dialogEvents: AlloyEve
     },
     dialogEvents,
     eventOrder: {
-      [SystemEvents.receive()]: [ 'reflecting', 'receiving' ],
+      [SystemEvents.receive()]: [ Reflecting.name(), Receiving.name() ],
```

Comment:
Why update these 2 and not the same ones below? If you're changing one, please change the others for consistency (ideally we should do this in all locations used in silver, but that's out of scope).

Reply:
Fixed now